app/src/main/res/layout/patientsummary_drug_item_content.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We are using `ConstraintLayout`s for any new UIs AFAIK. It's fine if the IDs being references between view since this is a single item view. I think it shouldn't be a problem 🤔

Reply:
`ConstraintLayout` for any new UI? That does not sound like a good idea to me. I will talk about this at some point.

But for now I guess, this PR is fine. I will accept it.